period/flag.go

```diff
@@ -0,0 +1,14 @@
+package period
+
+func (period *Period) Set(p string) error {
+	p2, err := Parse(p)
+	if err != nil {
+		return err
+	}
+	*period = p2
+	return nil
+}
+
+func (period Period) Type() string {
```

Comment:
Please would you comment this method; it's not obvious why it's needed otherwise. eg

```
// Type is for compatibility with the spf13/pflag library.
```

Reply:
Done!